playbooks/installer/ospd/main.yml

```diff
@@ -19,9 +19,17 @@
   hosts: undercloud
   gather_facts: no
   become: yes
-  become_user: "{{ installer.user.name }}"
+  become_method: sudo
   roles:
-      - {role: installer/ospd/images}
+      - role: installer/ospd/images
+        task: "{{ installer.images.task }}"
+        work_path: "{%- if installer.images.task == 'import' %}{{ '/home/{{ installer.user.name }}' }}{% endif %}"
+        ospd_version: "{{ installer.product.core.version }}"
```

Comment:
this is actually core (packstack version) so either you need `installer.product.version` or you need to change the name to `core_version`